app/controllers/pokemons_controller.rb

```diff
@@ -0,0 +1,41 @@
+class PokemonsController < ApplicationController
+
+  def new
+    @pokemon = Pokemon.new
+  end
+
+  def create
+    @pokemon = Pokemon.new
+    @pokemon.name = params[:pokemon][:name]
+    @pokemon.health = 100
+    @pokemon.level = 1
+    @pokemon.trainer_id = current_trainer.id
+    if @pokemon.save
+      redirect_to trainer_path(current_trainer.id)
+    else
+      if @pokemon.errors.any?
+        render 'new'
```

Comment:
You can add the error flashes here so the user gets feedback for what validations failed